pkg/terraform/stages/nutanix/stages.go

```diff
@@ -0,0 +1,14 @@
+package nutanix
+
+import (
+	"github.com/openshift/installer/pkg/terraform"
+	"github.com/openshift/installer/pkg/terraform/providers"
+	"github.com/openshift/installer/pkg/terraform/stages"
+)
+
+// PlatformStages are the stages to run to provision the infrastructure in Nutanix.
+var PlatformStages = []terraform.Stage{
+	stages.NewStage("nutanix", "pre-bootstrap", []providers.Provider{providers.Nutanix}),
+	stages.NewStage("nutanix", "bootstrap", []providers.Provider{providers.Nutanix}, stages.WithNormalBootstrapDestroy()),
+	stages.NewStage("nutanix", "master", []providers.Provider{providers.Nutanix}),
```

Comment:
Is there any technical reason why the "pre-bootstrap" and the "master" stages cannot be combined into a single stage which is run prior to the "bootstrap" stage? For example, will the nutanix terraform provider wait for the masters to fully ignite before the provider deems that master VMs to have been successfully created?

Reply:
The masters nodes get created after the bootstrap node gets created. The three stages are reflective of phases of the install pre-bootstrap(create images on Prism Central), bootstrap (provision bootstrap node), master (provision master nodes).

Reply:
My question was whether there is any technical reason why the masters could not be created **before** the bootstrap node is created.

Reply:
None really, added a change to merge those two. See if you like it more this way.